src/utils/fetchClient.ts

```diff
@@ -0,0 +1,41 @@
+const BASE_URL = 'https://mate.academy/students-api';
+
+function wait(delay: number) {
+  return new Promise(resolve => {
+    setTimeout(resolve, delay);
+  });
+}
+
+type RequestMethod = 'GET' | 'POST' | 'PATCH' | 'DELETE';
+
+function request<T>(
+  url: string,
+  method: RequestMethod = 'GET',
+  data: any = null,
```

Comment:
Please fix this warning
![image](https://user-images.githubusercontent.com/98764641/224671350-b4a9cd31-d5b0-42b3-8bd5-b3ed634a34c9.png)